src/repos.rs

```diff
@@ -0,0 +1,152 @@
+use std::collections::HashSet;
+
+use colored::Colorize;
+
+use crate::{Bootstrap, Repository};
+
+/// Count the number of non-archived repos with given visibility
+/// that have a certain security property set to "enabled"
+fn count_repos_security_property_enabled(
```

Comment:
I think we should make this more general (ie not related to a security property). That will encourage more reuse.

I also think we may want to support key blinding so I would engineer this with that in-mind :)